### PR TITLE
UPSTREAM: 86177: make test framework easier to re-use

### PR DESCRIPTION
**Pull request by deads2k**
baseline to use to add more test error handling.  I'd like oc inspect output on all involved namespaces and all clusteroperators

### CODE REVIEW

File: vendor/k8s.io/kubernetes/test/e2e/framework/framework.go [hunk 1]
```diff
@@ -31,6 +31,8 @@ import (
 	"sync"
 	"time"
 
+	"k8s.io/apimachinery/pkg/runtime"
```
Comment (mfojtik) [inline vendor/k8s.io/kubernetes/test/e2e/framework/framework.go]: nit: move down

Comment (deads2k) [reply vendor/k8s.io/kubernetes/test/e2e/framework/framework.go]: > nit: move down

it's a pick from upstream

**Pull request merged**